scripts/build_datapackage.py

```diff
@@ -242,21 +242,52 @@ def parametrize_sequences(edp, ts, filters):
     return edp
 
 
-def load_additional_scalars(scalars):
-    """Loads additional scalars like the emission limit."""
+def load_additional_scalars(scalars, filters):
+    """Loads additional scalars like the emission limit and filters by 'scenario_key'.
+    Emission limit is calculated by
+    emission_limit = (emissions_1990 - emissions_not_modeled) * (1 - emission_reduction_factor)"""
     # get electricity/gas relations and emission limit
     el_gas_rel = scalars.loc[scalars.var_name == EL_GAS_RELATION]
-    emissions = scalars.loc[scalars.var_name == EMISSION_LIMIT]
+    emissions = scalars.loc[scalars.carrier == EMISSION]
+
     # get `output_parameters` of backpressure components as they are not take into
     # consideration in oemof.tabular so far. They are added to the components' output flow towards
     # the heat bus in script `optimize.py`.
     bpchp_out = scalars.loc[
         (scalars.tech == "bpchp") & (scalars.var_name == "output_parameters")
     ]
+
     # save all dataframes in one file
     df = pd.concat([el_gas_rel, emissions, bpchp_out])
 
-    return df
+    # filter by 'scenario_key'
+    for id, filt in filters.items():
+        filtered = df.copy()
+        for key, value in filt.items():
+            filtered = filter_df(filtered, key, value)
+
+    # calculate emission limit and add to data frame
+    filtered_ = filtered.copy().set_index("var_name")
+    emission_limit = (
```

Comment:
This is the actual calculation. Could be a function `calculate_emission_limit` for better readability.